EasyPost.Tests.VB/EasyPost.Tests.VB.vbproj

```diff
@@ -10,9 +10,10 @@
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0"/>
-      <PackageReference Include="MSTest.TestAdapter" Version="2.2.8"/>
-      <PackageReference Include="MSTest.TestFramework" Version="2.2.8"/>
+      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+      <PackageReference Include="coverlet.collector" Version="1.2.0" />
+      <PackageReference Include="xunit" Version="2.4.2" />
+      <PackageReference Include="xunit.runner.visualstudio" Version="2.4.2" />
       <PackageReference Include="SecurityCodeScan.VS2019" Version="[5.0.0, 6.0.0)">
```

Comment:
Issue: These should all be pinned versions

Question: Why was `Microsoft.NET.Test.Sdk` downgraded?

Reply:
It wasn't? The same version is used on master
https://github.com/EasyPost/easypost-csharp/blob/69f918c4c00a4929ac578ea835fdb8e24425561e/EasyPost.Tests/EasyPost.Tests.csproj#L22

Reply:
This was bumped back after my question.